### PR TITLE
Updated test project to use latest version of Xunit

### PR DESCRIPTION
**Pull request by 1TheMuffinMan**
Also updated the build definition to use latest version of mono and Xunit test runner.

### CODE REVIEW

**Comment by MuffinMan10110** [comment]
Woohoo! 🎉 

@bhelx That took some effort, but finally got the tests to pass when running under Mono.

**Comment by bhelx** [comment]
Wow great job! I've been dreading that update.

**Comment by MuffinMan10110** [comment]
Thank's @bhelx 

What are your thoughts on merging this? I was thinking we could merge this and then I can submit a PR for the async/await feature just to keep the PR's small since so many files changed in this one.

**Comment by bhelx** [comment]
hey @1TheMuffinMan

I've been preoccupied with the other libraries this week. I'm going to give this a review this afternoon. I'm thinking it should be a quick merge.

File: Library/Invoice.cs [hunk 1]
```diff
@@ -104,6 +105,16 @@ public byte[] GetPdf(string acceptLanguage = "en-US")
         }
 
         /// <summary>
+        /// Returns a PDF representation of an invoice
+        /// </summary>
+        /// <param name="acceptLanguage">Language for invoice, defaults to en-US.</param>
+        /// <returns></returns>
+        public async Task<byte[]> GetPdfAsync(string acceptLanguage = "en-US")
```
Comment (bhelx) [inline Library/Invoice.cs]: Did this async stuff sneak in here?

Comment (1TheMuffinMan) [inline Library/Invoice.cs]: It appears that the commit I was expecting to release as a pull request after the one for updating the tests snuck into this PR somehow. The sneaky commit is in a finished state and only has 5 files changed which are adding additional endpoints. 

Would you prefer that I commit a revert and submit a new PR? I have no problem either way.

Thanks.

Comment (bhelx) [inline Library/Invoice.cs]: Yeah, hate to be a bother but I'd prefer for it to just have the commits intended. Keeps the history clean.

Comment (1TheMuffinMan) [inline Library/Invoice.cs]: No problem at all. I'll see if I can commit a revert tonight.

Comment (1TheMuffinMan) [inline Library/Invoice.cs]: I was able to get that revert in. Please let me know if there are any other items I can address. Thanks.

**Comment by bhelx** [comment]
Okay, I think this is looking good now :+1:

Thanks for cleaning it up!

**Pull request merged**